pyspark/docs/doc-source-gen.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+isSphinxInstalled=$(pip show Sphinx &>/dev/null; echo $?)
+if [ ! $isSphinxInstalled -eq 0 ]; then
+ pip install -U Sphinx
+fi
+
+isSphinxInstalled=$(pip show Sphinx &>/dev/null; echo $?)
+if [ ! $isSphinxInstalled -eq 0 ]; then
+ echo "Please install Sphinx"
+ exit 1
+fi
+
+isPy4jInstalled=$(pip show Py4j &>/dev/null; echo $?)
+if [ ! $isPy4jInstalled -eq 0 ]; then
+ pip install -U Py4j
+fi
+
+isPy4jInstalled=$(pip show Py4j &>/dev/null; echo $?)
+if [ ! $isPy4jInstalled -eq 0 ]; then
+ echo "Please install Py4j"
+ exit 1
+fi
+
+sphinx-apidoc -F -f -a -H BigDL -A Intel -o ./ ../dl
+
+PYSPARK=$SPARK_HOME'/python/lib/pyspark.zip'
+if [ ! -f "$PYSPARK" ]; then
+  echo "pyspark.zip for spark is not found."
```

Comment:
Cannot found SPARK_HOME . please set SPARK_HOME first.

and found pyspark.zip dynamically instead of hard coding.